### DISCUSSION TITLE
tool to design a database and export script or cmd command to doctrine

### DISCUSSION THREAD

**Discussion by wnunezc** [symfony/symfony#44560] [Q&A]
someone recommend me some tool to design a database then to carry them doctrine and generate them, like generated script or cmd to execute???

**Comment by GromNaN** [comment]
My team uses Skipper since 2015. It allows to design the relational database and export to Doctrine Entities. The reverse is not possible, all updates have to be done in Skipper. https://www.skipper18.com/

**Reply by wnunezc** [reply]
I had a stupid idea ok ...

![Captura de pantalla 2021-12-10 135832](https://user-images.githubusercontent.com/1207306/145631839-bd277494-7874-4f4e-9478-83b244ea12d8.jpg)

currently I do the schema layouts manually (like the image I show you, i pick then from internet), and then I have to go to the console to create the entities one by one (no matter how many they are) using the command `php bin\console make:entity` (i whan to avoid it) and filling the data manually (and avoid it) this is supposed to then generate the necessary files and directories in synfony which are the entities files used by symfonhy and doctrine.

I thought that some tool could do all the manual part starting from the design i make.

![Captura de pantalla 2021-12-10 135406](https://user-images.githubusercontent.com/1207306/145632333-1af4246a-5cd2-411f-80ea-25eb47084bd4.jpg)

the use of doctrine is only to do the update later the database schema... but this need the entities files created, that is that i whan to avoid to do manualy...

I've already read about Skipper but pay and have no budget to buy this software, nor is whether entities created by this software are compatible with the command: `php bin/console doctrine:schema:update --force`
